manager/apis/app/v1alpha1/m4dapplication_types.go

```diff
@@ -57,11 +58,17 @@ type ExternalDataContext struct {
 	// +optional
 	Recurrence RecurrenceType `json:"recurrence,omitempty"`
 }
+*/
```

Comment:
Is RecurrenceType commented out? Why?

Reply:
No, it's not.  The entire ExternalDataContext is commented out.  I will remove it.  Recurrence is in DataContext now.